rviz_default_plugins/CMakeLists.txt

```diff
@@ -86,29 +86,19 @@ set(rviz_default_plugins_headers_to_moc
   include/rviz_default_plugins/displays/laser_scan/laser_scan_display.hpp
   include/rviz_default_plugins/displays/map/map_display.hpp
   include/rviz_default_plugins/displays/marker/marker_common.hpp
-  include/rviz_default_plugins/displays/marker/marker_display.hpp
```

Comment:
Why were some files removed from the moc list? Is it because they don't need to be [pre-processed by moc](https://doc.qt.io/qt-5/moc.html)?

Reply:
> Why were some files removed from the moc list? Is it because they don't need to be [pre-processed by moc](https://doc.qt.io/qt-5/moc.html)?

Yeah, that's exactly right.  If you run the `moc` generation process by hand, and there is no `Q_OBJECT` or `Q_GADGET` in the header file, then moc prints a warning to stderr.  So I removed those.